crates/matrix-sdk-indexeddb/src/crypto_store/mod.rs

```diff
@@ -159,6 +159,81 @@ impl From<IndexeddbCryptoStoreError> for CryptoStoreError {
 
 type Result<A, E = IndexeddbCryptoStoreError> = std::result::Result<A, E>;
 
+/// Defines an operation to perform on the database
```

Comment:
(uber nit, for consistency:)

```suggestion
/// Defines an operation to perform on the database.
```